SeleniumPracticalExercise/PageObjects/AddEmployeePage.cs

```diff
@@ -31,9 +31,10 @@ public void SetEnterNames(string firstname, string lastname)
         }
 
         /// <summary>
-        /// get Employee Id value & returns the Id #
+        /// get Employee Id value & returns Id #
```

Comment:
This can be simply, "Returns the employee ID." Don't worry about the "get" portion of the description... just describe the overall goal of the method as simply as you can.